src/flux/stores/database-transaction.coffee

```diff
@@ -203,7 +203,7 @@ class DatabaseTransaction
         if joinedModels
           for joined in joinedModels
             joinMarks.push('(?,?)')
-            joinedValues.push(model.id, joined.id)
+            joinedValues.push(model.id, joined.queryableValue())
```

Comment:
currently only category implements this `queryableValue`, maybe it should be part of `Model` base class? Also still not entirely convinced about the name